src/platform/tests/TestConfigurationMgr.cpp

```diff
@@ -83,8 +83,9 @@ TEST_F(TestConfigurationMgr, SerialNumber)
 
     char buf[64];
     const char * serialNumber = "89051AAZZ236";
+    chip::Span<const char> serial_number(serialNumber.Value().c_str(), serialNumber.Value().size());
```

Comment:
ChrSpan, follow naming conventions.

But also, this code can't compile, right?  What you want is:
```
CharSpan serialNumber = 89051AAZZ236"_span;
```